default/col_buildings.txt

```diff
@@ -184,7 +184,7 @@ BuildingType
             activation = Turn low = Source.CreationTurn + 2
             effects = Destroy
     ]
-    icon = "icons/building/generic_building.png"
+    icon = "icons/species/abaddonnian.png"
```

Comment:
Actually, thinking about it more I really disagree. I used the species icon because it's the most appropriate and ties in with other effects giving a good visual key, if at some point we want to change the species icons we need to change one art file, but if e duplicate that file in multiple places then multiple files, and references to those files, need to change.